### DISCUSSION TITLE
Incorrect data of github repositories

### DISCUSSION THREAD

**Discussion by ArjunSharda** [community/community#18320] [Projects and Issues] [answered]
For me, it says I have 28 repositories although I have 14 public and 1 private. Video: https://www.screencast.com/t/M1TFDqGS

My github: https://www.github.com/ajsharda17

**Comment by zynowhere** [comment]
I see the correct numbers and
What is the achievement YOLO?
![sp220611_123159.png](https://tva1.sinaimg.cn/large/a12a0312ly1h3473cp5g0j20my06j766.jpg)

**Reply by ArjunSharda** [reply]
Only happening for me and yolo is part of a new achievement system

**Comment by DavidYang2149** [comment]
I guess it's browser cache.

Can you hard refresh browser and clear cache?

**Reply by ArjunSharda** [reply]
Will try

**Reply by ArjunSharda** [reply]
Nope, doesn't work

**Comment by ArjunSharda** [comment, marked as answer]
Fixed after I made a new repository
